source/features/contact/ContactPage.module.css

```
@@ -18,7 +18,7 @@
   margin-bottom: 2rem;
   max-width: 500px;
   line-height: 1.7;
-  margin: 0 auto 2rem auto; 
+  margin: 0 0 2rem;
```

Stylelint (17.4.0)

[error] Unexpected shorthand "margin" after "margin-bottom" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)